corehq/apps/data_interfaces/models.py

```diff
@@ -165,276 +165,6 @@ def references_parent_case(self):
 
         return False
 
-    @classmethod
-    def get_referenced_app_form_pairs_from_sms_surveys(cls, domain):
```

Comment:
first thought:

> wtf? why are you changing data_interfaces code?

realization:

> omg. the reaches of our features...

Reply:
The SMS code is only about a year old: https://github.com/dimagi/commcare-hq/pull/21634/

That might have been a good time to remove the exchange, instead of doing that work. (And I reviewed that PR, so I definitely could have brought it up and the time but didn't think of it.)